func_tests/test_client.py

```diff
@@ -98,6 +98,31 @@ def test_batch_change_review_process_reject(vinyldns_test_context):
     assert completed_bc.review_comment == 'cannot create the zone'
 
 
+def test_batch_change_review_process_cancel(vinyldns_test_context):
+    changes = [
+        AddRecord('test-approve-success.not.loaded.', RecordType.A, 200,
+                  AData("4.3.2.1"))
+    ]
+
+    bc = vinyldns_test_context.client.create_batch_change(
+        BatchChangeRequest(changes, 'comments', vinyldns_test_context.group.id)
+    )
+
+    assert bc.status == 'PendingReview'
+    assert bc.approval_status == 'PendingReview'
+    assert len(bc.changes[0].validation_errors) == 1
+
+    vinyldns_test_context.support_client.cancel_batch_change(bc.id)
```

Comment:
This should be `client` rather than `support_client`. I forgot to mention this, but only the user who made the batch change can cancel it. I'm not sure why the Python 2.7 version passed this test though.

Reply:
Still receiving the same error :disappointed: